.cp2k/Makefile

```diff
@@ -107,27 +175,22 @@ OBJ_SRC_FILES  = $(shell cd $(SRCDIR); find . ! -name "dbcsr_api_c.F" -name "*.F
 OBJ_SRC_FILES += $(shell cd $(SRCDIR); find . -name "*.c")
 
 # if compiling with GPU acceleration
-ifneq ($(NVCC),)
+ifneq ($(ACC),)
   # All *.cpp files belong to the accelerator backend
   OBJ_SRC_FILES += $(shell cd $(SRCDIR); find . ! -name "acc_cuda.cpp" ! -name "acc_hip.cpp" -name "*.cpp")
   # if compiling with nvcc
-  ifneq (,$(findstring nvcc,$(NVCC)))
+  ifneq (,$(findstring nvcc,$(ACC)))
     OBJ_SRC_FILES += $(LIBSMM_ACC_DIR)/../cuda/acc_cuda.cpp
     # Exclude autotuning files
     OBJ_SRC_FILES += $(shell cd $(SRCDIR);  find . ! -name "tune_*_exe*_part*.cu" ! -name "tune_*_exe*_main*.cu"  -name "*.cu")
   # if compiling with hipcc
-  else ifneq (,$(findstring hipcc,$(NVCC)))
+  else ifneq (,$(findstring hipcc,$(ACC)))
     OBJ_SRC_FILES += $(LIBSMM_ACC_DIR)/../hip/acc_hip.cpp
     # Exclude autotuning files
     OBJ_SRC_FILES += $(shell cd $(SRCDIR);  find . ! -name "tune_*_exe*_part*.cpp" ! -name "tune_*_exe*_main*.cpp"  -name "*.cpp")
```

Comment:
I've just tested compilation of CP2K with HIP-accelerated DBCSR and noticed we should delete these 2 lines.
Indeed, this adds some `.cpp` files to `OBJ_SRC_FILES` that were already added in line 180. This leads to errors in `makedep` when compiling, since source files appead twice in the list.